scalafmt-tests/src/test/resources/test/Enum.stat

```diff
@@ -0,0 +1,51 @@
+runner.dialect = dotty
+<<< enum
+enum Color {
+case Red, Green, Blue
+}
+>>>
+enum Color {
+  case Red, Green, Blue
+}
+<<< parameterized enum
+enum Color (val rgb: Int) {
+case Red   extends Color(0xff0000)
```

Comment:
Could you add some cases that a longer and might break the line? You can add an additional setting to make the max column shorter.

Reply:
Actually, this is interesting. So enums are represented as cases, which means the formatter might be looking at them as match cases, since there was nothing else before. Do we need to modify the processing of KwCase as well? At least add tests which will make enum cases look very different from match cases and show that it still works and uses the right amount of indentation.